### PR TITLE
Populate ganache chain before running tests

### PR DESCRIPTION
**Pull request by robert-e-davidson3**
Also made it optional to write addresses and such to disk since it's unnecessary and messy due to those files being checked into git.

### CODE REVIEW

File: packages/augur-test/src/scripts/GenerateGanacheSeed.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    leveledDB.createReadStream({
+      keyAsBuffer: false,
+      valueAsBuffer: false,
+    }).on("data", (data: LevelDBRow) => {
+      payload.push({
+        type: "put",
+        ...data,
+      });
+    }).on("error", (err: Error) => {
+        console.log("Oh my!", err);
```
Comment (nuevoalex) [inline packages/augur-test/src/scripts/GenerateGanacheSeed.ts:90]: 👍

**Review by nuevoalex — APPROVED** [review]
LGTM

File: packages/augur-test/src/scripts/GenerateGanacheSeed.ts (hunk 1 above)
Comment (robert-e-davidson3) [reply packages/augur-test/src/scripts/GenerateGanacheSeed.ts:90]: Credit goes to @justinbarry :)

**Comment by robert-e-davidson3** [comment]
Stole pieces of https://github.com/AugurProject/augur/pull/2242 to make this PR better.

**Pull request merged**